### PR TITLE
Add a paragraph about tuning couchjs

### PR DESCRIPTION
**Pull request by iblislin**
xref: apache/couchdb#1670

### CODE REVIEW

File: src/config/query-servers.rst [hunk 1]
```diff
@@ -59,6 +59,15 @@ Query Servers Definition
         javascript = /usr/bin/couchjs /usr/share/couchdb/server/main.js
         coffeescript = /usr/bin/couchjs /usr/share/couchdb/server/main-coffee.js
 
+    By default, ``couchjs`` limits the max runtime allocation in 64MiB.
```
Comment (flimzy) [inline src/config/query-servers.rst]: Should be "to 64MiB".

Comment (iblis17) [reply src/config/query-servers.rst]: fix

File: src/config/query-servers.rst [hunk 2]
```diff
@@ -59,6 +59,15 @@ Query Servers Definition
         javascript = /usr/bin/couchjs /usr/share/couchdb/server/main.js
         coffeescript = /usr/bin/couchjs /usr/share/couchdb/server/main-coffee.js
 
+    By default, ``couchjs`` limits the max runtime allocation to 64MiB.
+    If you run into out of memory issue in your ddoc functions,
+    you can adjust the memory limitation::
+
+        [query_servers]
+        javascript = /usr/bin/couchjs -S 536870912 /usr/share/couchdb/server/main.js ; 512 MiB
```
Comment (wohali) [inline src/config/query-servers.rst]: I don't believe you can use `;` as a comment character in the ini file.

Can you move the comment into the previous paragraph instead, so people cutting and pasting this snippet won't break their installations? Thanks!

Comment (iblis17) [reply src/config/query-servers.rst]: I think `;` works correctly in my couchdb config file.
And there is another exaple: 
https://github.com/apache/couchdb/blob/master/test/random_port.ini

Comment (iblis17) [reply src/config/query-servers.rst]: https://github.com/apache/couchdb/blob/master/rel/overlay/etc/local.ini#L8

**Pull request merged**